Source/DiabloUI/selconn.cpp

```diff
@@ -112,7 +112,8 @@ void SelconnFocus(int value)
 	}
 
 	strncpy(selconn_MaxPlayers, fmt::format(_("Players Supported: {:d}"), players).c_str(), sizeof(selconn_MaxPlayers));

```

Comment:
Shoudn't we be able to remove this string copy and instead use the string directly as the input?

Reply:
Not sure what you mean exactly but sounds like something that should go into a separate refactoring.

Reply:
Arh my mistake, I thought this was the text that was being wordwrapped on the next line. But it's being copied to `selconn_MaxPlayers` not `selconn_Description`.